### ISSUE TITLE
Lecture_01_What_is_SYCL should show some SYCL code

### ISSUE THREAD

**Issue by keryell** [codeplaysoftware/syclacademy#79]
`Lecture_01_What_is_SYCL` dives into various way of compiling SYCL and other tool architectures.
So after the slide 3 there should be a minimalistic but full SYCL code sample to show how simple and great SYCL is.
This will give some motivation for the 15 on-going hours. :-)

**Comment by fraggamuffin** [comment]
Agreed, actually I was thinking this might be the place to do a live demo of multi-targeted SYCL code:
https://github.com/codeplaysoftware/syclacademy/issues/98
